pkg/operation/botanist/component/dependencywatchdog/bootstrap.go

```diff
@@ -331,19 +328,33 @@ func (b *bootstrapper) getLabels() map[string]string {
 	return map[string]string{v1beta1constants.LabelApp: b.name()}
 }
 
-func (b *bootstrapper) clusterRoleRules() []rbacv1.PolicyRule {
+func (b *bootstrapper) roleRules() []rbacv1.PolicyRule {
 	switch b.values.Role {
-	case RoleEndpoint:
+	case RoleWeed:
 		return []rbacv1.PolicyRule{
 			{
 				APIGroups: []string{""},
-				Resources: []string{"endpoints"},
-				Verbs:     []string{"get", "list", "watch"},
+				Resources: []string{"endpoints", "events"},
```

Comment:
there is already a cluster role for `endpoints` for the weeder. Is there a reason to also add a `role` for the same resource?

Reply:
yes there should be no reason , I'll remove it.